store/continents.js

```diff
@@ -0,0 +1,70 @@
+
+import axios from 'axios'
+
+const CONTINENTS_URL = 'https://cdn.rawgit.com/OGCMetOceanDWG/wmo-ra/master/wmo-ra.geojson'

```

Comment:
I'd recommend putting that url in `.env.sample` and referencing that env variable.

```suggestion
const CONTINENTS_URL = process.env.CONTINENTS_URL
```

Also, this geojson file is not a list of continents. They're WMO regions. Not sure if you want to rename the file/variables to avoid any confusion.

Otherwise, excellent work here with using the vuex store.

Reply:
Thanks, that makes a lot more sense actually that those are WMO regions.